test/nat46x64/test.sh

```diff
@@ -1,423 +1,411 @@
 #!/usr/bin/env bash
 
 PS4='+[\t] '
```

Comment:
So heads up by default these timestamps won't show up when you run the script, unless you set the env var `V=1` like `V=1 sudo -E ./test/nat46x64/test.sh`. I think this is fine as a tradeoff, in GitHub actions we have access to timestamps anyway without enabling this:

![image](https://github.com/cilium/cilium/assets/1243336/316d335a-dbed-48a8-9798-e280eccbb73d)